pallets/committee/src/types.rs

```diff
@@ -208,3 +214,34 @@ impl<
         ))
     }
 }
+
+/// Ensure committee member
+pub struct EnsureMember<T>(sp_std::marker::PhantomData<T>);
+
+impl<
+        O: Into<Result<RawOrigin<<T as frame_system::Config>::AccountId>, O>>
+            + From<RawOrigin<<T as frame_system::Config>::AccountId>>
+            + Clone,
+        T: Config,
+    > EnsureOrigin<O> for EnsureMember<T>
+{
+    type Success = <T as frame_system::Config>::AccountId;
+    fn try_origin(o: O) -> Result<Self::Success, O> {
+        let origin = o.clone().into()?;
+        match origin {
+            RawOrigin::Signed(i) => {
+                if <Members<T>>::contains_key(i.clone()) {
```

Comment:
does `contains_key` require ownership of `i`, hence the clone?

Reply:
Oops, this is a nit caused the previous implementation